rust/trace/src/sys_tid.rs

```diff
@@ -86,5 +86,5 @@ pub fn current_tid() -> Result<u64, libc::c_int> {
         fn GetCurrentThreadId() -> libc::c_ulong;
     }
 
-    unsafe { Ok(GetCurrentThreadId() as u64) }
+    unsafe { Ok(u64::from(GetCurrentProcessId())) as u64 }
```

Comment:
This does not even compile. The fn in question returns a Result but you are forcibly casting it to a plain u64.

Reply:
This is also trying to call the wrong function now. `GetCurrentProcessId` is not available here, should be `GetCurrentThreadId` like it originally was.